mlflow/utils/cli_args.py

```diff
@@ -202,13 +202,13 @@ def _create_env_manager_option(help_string):
 )
 
 SERVE_ARTIFACTS = click.option(
-    "--serve-artifacts",
+    "--serve-artifacts/--no-serve-artifacts",
```

Comment:
Can we explain the behavior when `--no-serve-artifacts` is specified?

Reply:
Sure!